### PR TITLE
Edit the Access Controls getting started guide

### PR DESCRIPTION
**Pull request by ptgott**
1. Edit the introduction.

   - Remove unnecessary sentences
   - Move the "What is RBAC" text to the introduction so we describe RBAC before saying that this guide shows you how to use it.

2. Move the outline of RBAC components into a "How it works" section. This follows the structure of other how-to guides and adds some context by using prose.

3. Change the step-by-step instructions.

   - Make these first in the guide to keep with the conventions for our step-by-step guides.
   - Add steps to enroll two servers as Docker containers in order to demonstrate RBAC.

4. Add conceptual sections under the step-by-step instructions in a "Next steps" section. This follows the convention for our how-to guides.

   - Reformat "Maintaining your labeling strategy" to make it more compact.
   - Edit style in the use cases section to use headings and add a short introduction to each code block.

5. Remove the per-resource RBAC section. This section combines Database Access Controls and Kubernetes per-resource RBAC. We use "per-resource RBAC" to refer to Kubernetes resource RBAC. Instead, add a section that links to RBAC-related guides for each kind of resource.

### CODE REVIEW

**Comment by ptgott** [comment]
@mmcallister I've made suggestions in the RBAC introduction to follow the conventions for our how-to guides as well as include a full demo environment to show how label matching works. Let me know what you think!

File: docs/pages/admin-guides/access-controls/getting-started.mdx [hunk 1]
```diff
@@ -264,10 +385,9 @@ Set up comprehensive logging and alerting for access events, especially for sens
 Ensure that all users understand their access levels and the importance of adhering to security policies.
 Document your role configurations.
 
-
-## Next steps
+## Further reading
 
 - [Mapping SSO and local users traits with role templates](../access-controls/guides/role-templates.mdx)
```
Comment (mmcallister) [inline docs/pages/admin-guides/access-controls/getting-started.mdx:390]:
```suggestion
- [Mapping SSO and local users traits with role templates](../access-controls/guides/role-templates.mdx).
- [Add Labels to Resources](../management/admin/labels.mdx).
```

File: docs/pages/admin-guides/access-controls/getting-started.mdx [hunk 2]
```diff
@@ -264,10 +385,9 @@ Set up comprehensive logging and alerting for access events, especially for sens
...
 - [Access Control for Servers](../../enroll-resources/server-access/rbac.mdx)
```
Comment (mmcallister) [inline docs/pages/admin-guides/access-controls/getting-started.mdx:391]:
```suggestion
- [Access Control for Servers](../../enroll-resources/server-access/rbac.mdx).
```

File: docs/pages/admin-guides/access-controls/getting-started.mdx [hunk 3]
```diff
@@ -264,10 +385,9 @@ Set up comprehensive logging and alerting for access events, especially for sens
...
 - [Access Control for Kubernetes](../../enroll-resources/kubernetes-access/controls.mdx)
```
Comment (mmcallister) [inline docs/pages/admin-guides/access-controls/getting-started.mdx:392]:
```suggestion
- [Access Control for Kubernetes](../../enroll-resources/kubernetes-access/controls.mdx).
```

**Comment by mmcallister** [comment]
Looks great! Thanks for diving in @ptgott and for helping me better understand how to structure this. Not much to note, added a link to the labeling information page. Looks like we usually end 'further reading lines' with periods (should we standardize this? or do you think it looks better without them?)

**Review by mmcallister — APPROVED** [review]

**Pull request merged**